lib/lighthouse/letters_generator/service_error.rb

```diff
@@ -0,0 +1,24 @@
+module Lighthouse
+  module LettersGenerator
+    class ServiceError < StandardError
+      attr_accessor :title, :status, :message
+
+      # Expects a response in one of these formats:
+      #
+      #  { status: "", title: "", detail: "", type: "", instance: "" }
+      # OR 
+      #  { message: "" }
+      #
+      #  @exception Exception [Faraday::ClientError|Faraday::ServerErrror] the exception returned by Faraday middleware
+      def initialize(exception = nil) 
+        unless exception.nil?
+          r = exception.response
```

Comment:
This should probably be `r = exception.response[:body]`

Reply:
Oh shoot, I think I dropped this with the Git shenanigans. Thank you!